From 4df19d35188fe9e49eb720760b7aba9ebc3e80e1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sebastian=20Dr=C3=B6ge?= <sebastian@centricular.com>
Date: Wed, 24 Feb 2016 16:46:11 +0200
Subject: [PATCH 3/4] Disable ncrypt support

It's Windows Vista only and our mingw toolchain does not support it currently
either.
---
 lib/Makefile.am | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/lib/Makefile.am b/lib/Makefile.am
index 873a569..1ce9728 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -85,11 +85,11 @@ COBJECTS = gnutls_range.c gnutls_record.c \
 	safe-memfuncs.c inet_pton.c atfork.c atfork.h \
 	system-keys.h urls.c urls.h auto-verify.c
 
-if WINDOWS
-COBJECTS += system-keys-win.c
-else
+#if WINDOWS
+#COBJECTS += system-keys-win.c
+#else
 COBJECTS += system-keys-dummy.c
-endif
+#endif
 
 if ENABLE_SELF_CHECKS
 COBJECTS += crypto-selftests.c crypto-selftests-pk.c
-- 
2.7.0

